skimage/restoration/_denoise.py

```diff
@@ -400,7 +400,7 @@ def denoise_wavelet(img, sigma=None, wavelet='db1', mode='soft',
 
     Parameters
     ----------
-    img : ndarray (2D/3D) of ints, uints or floats
+    img : ndarray ([M[, N[, ...P]][, C]) of ints, uints or floats
```

Comment:
Sorry, actually this should be `(M, N[, ...P][, C])` if we declare 2+D support. Please, notice that `[, ]` mark _optional_ dimensions.